polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -1569,6 +1569,10 @@ ValueCategory MLIRScanner::VisitCastExpr(CastExpr *E) {
   case clang::CastKind::CK_UserDefinedConversion:
     return Visit(E->getSubExpr());
 
+  case clang::CastKind::CK_LValueBitCast: {
+    E->dump();
+    llvm_unreachable("LValue bitcast not handled yet\n");
```

Comment:
All ESIMD tests are failed due to this.

Reply:
I can look into this when I finish work on `fsycl-raise-host` option

Reply:
Thanks Victor!